plan/physical_plans.go

```diff
@@ -432,6 +432,43 @@ type Cache struct {
 	basePlan
 }
 
+func (p *PhysicalMergeJoin) tryConsumeOrder(prop *requiredProperty, eqCond *expression.ScalarFunction) *requiredProperty {
+	// TODO: We still can consume a partial sorted results somehow if main key matched.
+	// To do that, we need a Sort operator being able to do a secondary sort
+	if len(prop.props) != 1 || len(eqCond.GetArgs()) != 2 {
+		return prop
+	}
+
+	reqSortedColumn := prop.props[0].col
+	sortDesc := prop.props[0].desc
```

Comment:
Yes.That means it doesn't consume any order since we don't have desc implementation exactly.